src/main/java/pl/psnc/dei/service/BatchService.java

```diff
@@ -138,7 +139,14 @@ private String getTitle(String recordId) {
 					.filter(obj -> obj.get("dc:title") != null)
 					.findFirst();
 			if (title.isPresent()) {
-				return title.get().getAsObject().get("dc:title").getAsObject().get("@value").getAsString().value();
+				Object dcTitle = title.get().getAsObject().get("dc:title");
+				if (dcTitle instanceof JsonObject) {
```

Comment:
Don't check this using instanceof. Json classes have method for checking whether one is a JsonObject, JsonString and so on (isObject(), isString(), ...)

Reply:
fixed